README.md

```diff
@@ -29,14 +29,14 @@ const Quote = require('@editorjs/quote');
 
 ### Load from CDN
 
-You can load specific version of package from [jsDelivr CDN](https://www.jsdelivr.com/package/npm/@editorjs/quote).
+You can load specific version of package from [jsDelivr CDN](https://www.jsdelivr.com/package/npm/@itech-indrustries/quote).
 
-`https://cdn.jsdelivr.net/npm/@editorjs/quote@2.0.0`
+`https://cdn.jsdelivr.net/npm/@itech-indrustries/quote@latest`
```

Comment:
you should maintain editor-js references

Reply:
Resolved